### ISSUE TITLE
Make screenshots a required part of submission

### ISSUE THREAD

**Issue by rsgonzal** [Iridescent-CM/technovation-app#2850]
Currently able to submit without screenshots: 
https://technovation-qa.herokuapp.com/admin/team_submissions/best-app-by-testb

- Students should not be able to submit their app without screenshots
- Screenshots should count as part of % complete in the admin

**Comment by shaunxp20** [comment]
This one finally made it to QA, some QA steps have been added too! 😄 

Good catch w/ this bug, a sneaky one! 🕵️‍♀️ 🕵️  Looks like images/screenshots haven't been required since 2018!
The commit that made them not required: https://github.com/Iridescent-CM/technovation-app/commit/68e928d09

**Comment by ophelieh** [comment]
Checked that I couldn't submit without at least 2 images. Submission % went from 75% to 100% after I added the pictures.

It seems to take time to recognize that I have 2 pictures. I have to click on an item of the submission for it to register. Otherwise, I cannot click the submit. 

<img width="1440" alt="Screen Shot 2021-03-12 at 5 38 07 PM" src="https://user-images.githubusercontent.com/32206421/111009608-b829b080-8359-11eb-9f8b-f325ec224086.png">

Once I click on something from the submission list, then I was able to get the submit button. 

<img width="1440" alt="Screen Shot 2021-03-12 at 5 39 08 PM" src="https://user-images.githubusercontent.com/32206421/111009650-dc858d00-8359-11eb-8310-0851ab7a4deb.png">

Another thing to note is that I went back into my submission and remove one of the pictures, then directly clicked "View my submission", I was able to. When I got back to the submission page though, it did show me that my submission was no longer complete.  So if I left after getting on my submission page, I might not realize that my submission is no longer considered complete... 

<img width="1440" alt="Screen Shot 2021-03-12 at 5 40 11 PM" src="https://user-images.githubusercontent.com/32206421/111009701-00e16980-835a-11eb-9921-55e8e79ca12b.png">
<img width="1440" alt="Screen Shot 2021-03-12 at 5 40 25 PM" src="https://user-images.githubusercontent.com/32206421/111009724-0a6ad180-835a-11eb-9765-bf29c6032d69.png">

**Comment by nickiredescent** [comment]
Tested and can confirm that teams must submit at least 2 images and admins can see a % progress update.

However, other than the sidebar, there is no clear indication that a student must submit 2 images. I would recommend help text above the button:
![image](https://user-images.githubusercontent.com/45576078/111212396-c7dd0b00-858c-11eb-83f0-bd3e658fa140.png)

The text can be the same as what we have in the submission guidelines:

> 2-6 images that showcase your Technovation journey. They should help judges better understand your ideas.

**Comment by rsgonzal** [comment]
> However, other than the sidebar, there is no clear indication that a student must submit 2 images. I would recommend help text above the button:
> ![image](https://user-images.githubusercontent.com/45576078/111212396-c7dd0b00-858c-11eb-83f0-bd3e658fa140.png)

@shaunxp20 let's use the type of text from the header below. And I would actually make it clear that two are required "Upload 2-6 images that showcase your Technovation journey. They should help judges better understand your ideas. You must submit at least 2."
![Screen Shot 2021-03-15 at 3 37 59 PM](https://user-images.githubusercontent.com/22801026/111230395-ccadb900-85a4-11eb-975a-2b3cac990b6f.png)


> It seems to take time to recognize that I have 2 pictures. I have to click on an item of the submission for it to register. Otherwise, I cannot click the submit.

@shaunxp20 I've noticed this as well, but I figured maybe it was a limitation of the screens? Like ophelie recognized, you need to refresh or click out of the page for it to realize that the task is complete. Would be great if it would be automatic, but if not 🤷‍♀️ 

> Another thing to note is that I went back into my submission and remove one of the pictures, then directly clicked "View my submission", I was able to. When I got back to the submission page though, it did show me that my submission was no longer complete. So if I left after getting on my submission page, I might not realize that my submission is no longer considered complete...

@ophelieh  this is a problem throughout :( #2859

**Comment by shaunxp20** [comment]
Thanks for taking a look @ophelieh @nickiredescent! 🙌 🍰 🎈 

I added the new "You must submit at least 2 images..."  text.

I also made a fix regarding this issue:

> > It seems to take time to recognize that I have 2 pictures. I have to click on an item of the submission for it to register. Otherwise, I cannot click the submit.
> 
> @shaunxp20 I've noticed this as well, but I figured maybe it was a limitation of the screens? Like ophelie recognized, you need to refresh or click out of the page for it to realize that the task is complete. Would be great if it would be automatic, but if not 🤷‍♀️

I made it so that after uploading an image, the page will get refreshed (just tested it a little, seems to work okay); the issue is that the submit button needs to get refreshed somehow (refreshing the page, or visiting another page) for it to become enabled.

**Comment by ophelieh** [comment]
New text for images looks great! 

It looks like it now refreshes when you add a picture (AMAZING!!! Thanks Shaun!!! :) ) - the submit button was activated once I added two pictures. However, when I removed 1 picture (and was left with only 1), the submit button was still activated and I was able to go through the full submission process without any errors. Not sure how big of an issue this would be. _I guess when I download the incomplete submissions before and after April 19th I could look for missing pictures._

<img width="1440" alt="Screen Shot 2021-03-16 at 8 46 11 PM" src="https://user-images.githubusercontent.com/32206421/111402137-a701dc00-8698-11eb-924d-ad493bc75aca.png">

When I logged back in though, it showed my submission was incomplete - same for the admin side it went from 100% to 75%. 

<img width="1440" alt="Screen Shot 2021-03-16 at 8 46 43 PM" src="https://user-images.githubusercontent.com/32206421/111402181-b97c1580-8698-11eb-878d-9277aabb93ff.png">

**Comment by shaunxp20** [comment]
> However, when I removed 1 picture (and was left with only 1), the submit button was still activated and I was able to go through the full submission process without any errors. Not sure how big of an issue this would be. _I guess when I download the incomplete submissions before and after April 19th I could look for missing pictures._

Good catch! I made the same change for deleting images, so now it will trigger a page refresh so the submit will get updated. It's on QA, hopefully we should be good now. 🤞 😄 😀

**Comment by ophelieh** [comment]
Awesome! Seems to work as expected - when I have at least 2 pictures the submit button is activated automicatically and when I go down to one it deactivates automatically. 

Thanks :)

**Comment by rsgonzal** [comment]
Looks great!

**Comment by shaunxp20** [comment]
This one has been deployed, thanks everyone! 🍰 🍪 🍰 🍪
